src/core/binding.c

```diff
@@ -816,12 +825,10 @@ QuicBindingProcessStatelessOperation(
         CxPlatRandom(sizeof(uint8_t), &RandomValue);
         VerNeg->Unused = 0x7F & RandomValue;
 
-        memcpy(Buffer, &Binding->RandomReservedVersion, sizeof(uint32_t));
```

Comment:
Why? It makes it harder for the server to validate the client's packet. Also, Where should it go? First or last? In the case the app sets a list of supported versions, where should it go?

Reply:
Because it greases the feature (VN). You can put it where ever makes it easier. You can just always put it first and know the validation logic and just "make sure the first is a reserved value". We wouldn't care which one exactly.